azurelinuxagent/common/utils/textutil.py

```diff
@@ -398,35 +396,3 @@ def format_memory_value(unit, value):
         raise TypeError('Value must be convertible to a float')
 
     return int(value * units[unit])
-
-def str_to_encoded_ustr(s, encoding='utf-8'): # pylint: disable=C0103
```

Comment:
I remember I was using this function outside this PR too, so just keep an eye out to make sure all references of this are removed